Description: Upstream changes introduced in version 0.2.0-1nexenta3
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libbsd (0.2.0-1nexenta3) unstable; urgency=low
 .
   * Package for NCP4.
 .
 The person named in the Author field signed this changelog entry.
Author: Andrew Stormont (astormont) <andyjstormont@googlemail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- libbsd-0.2.0.orig/include/bsd/stdlib.h
+++ libbsd-0.2.0/include/bsd/stdlib.h
@@ -56,8 +56,10 @@ void arc4random_addrandom(u_char *dat, i
 
 int dehumanize_number(const char *str, int64_t *size);
 
+#ifndef __sun
 char *getprogname ();
 void setprogname (char *);
+#endif
 
 int heapsort (void *, size_t, size_t, int (*)(const void *, const void *));
 
--- libbsd-0.2.0.orig/src/progname.c
+++ libbsd-0.2.0/src/progname.c
@@ -28,6 +28,8 @@
   Rejected in glibc (http://sourceware.org/ml/libc-alpha/2006-03/msg00125.html)
 */
 
+#ifndef __sun
+
 #include <bsd/stdlib.h>
 
 static char *__progname = NULL;
@@ -42,4 +44,6 @@ void
 setprogname(char *new)
 {
 	__progname = new;
+
 }
+#endif
